AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -145,6 +149,10 @@ - (void)updateCurrentRangeWithMode:(ASLayoutRangeMode)rangeMode
     _didUpdateCurrentRange = YES;
```

Comment:
Does this preserve manual mode logic work if a controller is hidden deep in a stack (e.g. visibilityDistance 4), but one of its data models updates and triggers a slight height resize or an item to be added to a collection?  If that happened and it re-flows its ranges, would it then kick back into Minimum or Normal mode?  Even though it may seem minor, such behavior could result in really impactful memory problems depending on how an app listens to model updates.